src/Files.Uwp/DataModels/NavigationControlItems/DriveItem.cs

```diff
@@ -69,7 +69,11 @@ public ByteSize FreeSpace
         public ByteSize SpaceUsed
         {
             get => spaceUsed;
-            set => SetProperty(ref spaceUsed, value);
+            set
+            {
+                SetProperty(ref spaceUsed, value);
+                HoverDisplayText = (Path.Contains("?", StringComparison.Ordinal) ? Text : Path) + $"\n{"PropertiesDriveUsedSpace/Text".GetLocalized()} {spaceUsed.ToSizeString()}";
```

Comment:
Instead of including this in the setter, is it better practice to listen for the property change notification and to call a dedicated method for setting the HoverDisplayText?

Reply:
Sorry, I didn't know

Reply:
I'm not sure myself, maybe @d2dyno1 can giveus some direction.